### ISSUE TITLE
Generics day (2 iteration)

### ISSUE THREAD

**Issue by eloylp** [coruna-gophers/lobby#3]
Hi  gophers !!

After [our first generics session](https://github.com/coruna-gophers/lobby/issues/2) , @mmontes11 and myself thought that would be great to dedicate a second session, this time, with the already stable [Go 1.18 version](https://go.dev/doc/go1.18). Honestly, we were delighted about the current implementation and now we want to go more in depth with it.

The "soft schedule" could be:

* Review the [POC](https://github.com/coruna-gophers/generics-poc) already done in the first session and check some more basic examples (warm up).
* Find a real use case for implementing generics. Something helpful. We are not discarding modifying (and maybe contribute) to any other open source project.

If you were not present in the first generic events ... It doesn't matter !! in case its helpful, we can try to help you to catch up very quickly ! Just come and enjoy.

#### Event information

* Location: [Its green](https://itsgreen.es/)
* Time: Monday, 11th April 2022 - 8:00 UTC
* **Capacity**: This is an informal event, thus we don't have any kind of reservation at the proposed location. However, if you add a comment on this issue confirming your assistance, we can try to better organise this event in a best-effort way.

#### Confirmed attendees

* @eloylp
* @mmontes11 

**In order to attend** to this event, please, write a comment confirming your assistance on this issue, we will add you to the attendees list. In this way, we will be notified by Github and maybe better accommodate people :smiley:

**Comment by eloylp** [comment]
Some of you mentioned that using the @coruna-gophers handle is not sending notifications, we need to investigate why not :memo:  . In the meanwhile, pinging here all the people:

* @edusalguero 
* @martinbecerradev 
* @rafacouto 
* @zucchinidev

**Comment by eloylp** [comment]
All right, some weeks ago, we met at this event. In this second iteration we focused on doing some tests by creating the same binary tree implementation, one generic and other one non generic. Then [we tried](https://github.com/coruna-gophers/generics-poc/pull/1) to measure the performance of both implementations, as the current Go generics implementation seems has [some runtime implications](https://planetscale.com/blog/generics-can-make-your-go-code-slower).

Great session, as usual. Probably worth digging a bit more on generics! Closing this !
